content/en/user-guide/integrations/quarkus/index.md

```diff
@@ -0,0 +1,156 @@
+{{< command >}}
+$ ./mvnw clean package -Dnative.
+{{< /command >}}
+{{< /alert >}}
+
+
+## Supported extensions
+
+- [Lambda](https://docs.quarkiverse.io/quarkus-amazon-services/dev/amazon-lambda.html)
+- [S3](https://docs.quarkiverse.io/quarkus-amazon-services/dev/amazon-s3.html)
+- [SSM](https://docs.quarkiverse.io/quarkus-amazon-services/dev/amazon-ssm.html)
+- [SQS](https://docs.quarkiverse.io/quarkus-amazon-services/dev/amazon-sqs.html)
+- [SNS](https://docs.quarkiverse.io/quarkus-amazon-services/dev/amazon-sns.html)
+- [SES](https://docs.quarkiverse.io/quarkus-amazon-services/dev/amazon-ses.html)
+- [Secrets Manager](https://docs.quarkiverse.io/quarkus-amazon-services/dev/amazon-secretsmanager.html)
+- [KMS](https://docs.quarkiverse.io/quarkus-amazon-services/dev/amazon-kms.html)
+- [IAM](https://docs.quarkiverse.io/quarkus-amazon-services/dev/amazon-iam.html)
+
+## Configuration
+
+The Configuration properties are fixed at build time. All the other configuration properties are overridable at runtime.
+
+| Property                                   | Type                   | Default                              |
+|----------------------------------------------------------|------------------------|--------------------------------------|
+| `quarkus.aws.devservices.localstack.image-name`           | string                 | `localstack/localstack:3.0.1`            |
+| `quarkus.aws.devservices.localstack.init-scripts-folder`  | string                 |                                      |
+| `quarkus.aws.devservices.localstack.init-scripts-classpath`| string                 |                                      |
+| `quarkus.aws.devservices.localstack.init-completion-msg`  | string                 |                                      |
+| `quarkus.aws.devservices.localstack.container-properties` | `Map<String,String>`  |                                      |
+| `quarkus.aws.devservices.localstack.additional-services."additional-services".enabled` | boolean                |                                      |
+| `quarkus.aws.devservices.localstack.additional-services."additional-services".shared`  | boolean                | `false`                              |
+| `quarkus.aws.devservices.localstack.additional-services."additional-services".service-name` | string                 | `localstack`                         |
+| `quarkus.aws.devservices.localstack.additional-services."additional-services".container-properties` | `Map<String,String>`  |                                      |
+
+{{< alert title="Note" >}}
+- If `quarkus.aws.devservices.localstack.additional-services."additional-services".enabled` is set to `true` and the endpoint-override is not configured, LocalStack will be started and utilized instead of the provided configuration. For all services excluding Cognito, LocalStack will function as the core cloud emulator.  In the case of Cognito, the emulation/mocking will be done by Moto.
```

Comment:
Not sure whether the additional information related to Moto is required - seems like an implementation detail and not really necessary information for the user, wdyt? 

Also, curious why we're using Cognito from moto - presumably to allow users to run local tests even with the community version..? If that is the case, then we could maybe mention that Moto provides a basic version, whereas the full-fledged features are provided when using LS Pro?

Reply:
> Also, curious why we're using Cognito from moto

We aren't recommending it 😅 Quarkus uses Moto for Cognito for some reason, and not LocalStack. It maybe because Cognito falls under LocalStack Pro.